docs/download/index.md

```diff
@@ -15,15 +15,11 @@ License & terms of use: http://www.unicode.org/copyright.html
 If you want to use ICU (as opposed to developing it), it is recommended that you download an official packaged version of the ICU source code. These versions are tested more thoroughly than day-to-day development builds of the system, and they are packaged in zip and tar files for convenient download. Here are several recent releases of ICU that are available:
 
 ## Release Candidate
```

Comment:
Oups
"Release"
?

Reply:
This page is the landing page for ICU downloading in general. Once the final release is made, do we remove the "Release Canadidate" section until the next version's RC is created?